astria-rpc/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Emit the `cargo:rerun-if-changed=proto/` instruction to force a rebuild: https://doc.rust-lang.org/cargo/reference/build-scripts.html#rerun-if-changed